task_geo/data_sources/demographics/us_census/us_census.py

```diff
@@ -37,22 +37,13 @@ def us_census_connector():
 
 
     Description:
-        - Opens the zip file URL and extracts the correct CSV
-        - Correct CSV: ACS 5Y Statistics
+        - Read the dataset in JSON format
 
     Returns:
         data (DataFrame with CSV Data)
     """
 
-    urllib.request.urlretrieve(url, "uscensus.zip")
-
-    with zipfile.ZipFile("uscensus.zip") as myzip:
-
-        listFiles = myzip.namelist()
-
-        myzip.extract(listFiles[5])
-        data = pd.read_csv(listFiles[5], low_memory=False)
-
+    data = pd.read_json(URL)
```

Comment:
You don't need to assign the dataframe to a variable and then return it, you can return it directly:
```python
return pd.read_json(URL)
```